client/html/src/Client/Html/Catalog/Product/Standard.php

```diff
@@ -452,15 +464,15 @@ public function addData( \Aimeos\MW\View\Iface $view, array &$tags = [], &$expir
 		 * @see client/html/catalog/stock/url/config
 		 */
 
-		if ( !empty( $products ) && (bool)$config->get( 'client/html/catalog/product/stock/enable', true ) === true ) {
+		if( !empty( $products ) && (bool) $config->get( 'client/html/catalog/product/stock/enable', true ) === true ) {
 			$view->itemsStockUrl = $this->getStockUrl( $view, $products + $productItems );
 		}
 
 		// Delete cache when products are added or deleted even when in "tag-all" mode
 		$this->addMetaItems( $products + $productItems, $expire, $tags, ['product'] );
 
-		$view->listProductItems = $products;
-		$view->listProductTotal = count( $products );
+		$view->productItems = $products;
+		$view->productTotal = count( $products );
 		$view->itemsProductItems = $productItems;
```

Comment:
Please use `$view->productProductItems` because the prefix should be the same

Reply:
No, this refers to `itemsProductItems`. You should use:
```
$view->productItems = $products;
$view->productTotal = count( $products );
$view->productProductItems = $productItems;
```
so all view properties have the same prefix

Reply:
Oh my bad. I didn't quite understand the naming scheme, but I think i got it now.